samples/remote_host_attestation/README.md

```diff
@@ -0,0 +1,128 @@
+
+# The host verification sample
+
+- Demonstrates the attestation of a remote host
+- Requires only the `openenclave/host_verify.h` header, as well as some C standard header files
+- Also runs in simulation mode and when both BUILD_ENCLAVES and HAS_QUOTE_PROVIDER flags are turned off
+
+Prerequisites: you may want to read [Common Sample Information](../README.md#common-sample-information) before going further
+
+## About the host verification sample
+
+When a host is doing host verification, or remote host attestation to be exact, it means that it is trying to authenticate its hardware and software settings so that it can determine whether or not to trust the remote host.
+This sample does remote host attestation by taking two arguments:
+
+- An SGX report
+- An SGX certificate
+
+However, the settings vary from time to time, so here we do not provide a pair of sample report and certificate, where a user can generate them with `oecert`. See [here](../../tests/tools/oecert/README.md) for more details.
+
+In the main function, if it sees the atrribute `-r`, then it will continue to read the next argument to get the report and call `verify_report()` to verify the report.
+Likewise, if it sees the attribute `-c`, then it will continue to read the next argument to get the certificate and call `verify_cert()` to verify the certificate.
+There is also an `enclave_identity_verifier()` function, which is called by `verify_cert()` and shows the information of a certificate if you feed it to the program.
+
+## Build and run
+
+Open Enclave SDK supports building the sample on both Linux and Windows.
+Linux supports two types of build systems, GNU Make with `pkg-config` and CMake,
+while Windows supports only CMake.
+
+### Linux
+
+#### CMake
+
+This uses the CMake package provided by the Open Enclave SDK.
+
+```bash
+cd remote_host_attestation
+mkdir build && cd build
+cmake ..
+make run
+```
+
+#### GNU Make
+
+```bash
+cd remote_host_attestation
+make build
+make run
+```
+
+### Windows
+
+#### CMake
+
+```bash
+mkdir build && cd build
+cmake .. -G Ninja -DNUGET_PACKAGE_PATH=C:\oe_prereqs
+ninja
+ninja run
+```
+
+When you see the following message displayed on the screen, it means you have successfully run the sample.
+
+```bash
+Usage:
+  ./remote_host_attestation -r <report_file>
+  ./remote_host_attestation -c <certificate_file>}
+```
+
+## Build and run with LVI mitigation
```

Comment:
Host verification does not intersect with LVI. Please remove text related to lvi.